components/dot_menu/dot_menu.tsx

```diff
@@ -421,6 +446,19 @@ export class DotMenuClass extends React.PureComponent<Props, State> {
             </span>
         );
 
+        const fromWebhook = this.props.post.props?.from_webhook === 'true';
+        const fromBot = this.props.post.props?.from_bot === 'true';
+        const canPostBeForwarded = !(fromWebhook || fromBot || isSystemMessage);
+
+        const forwardPostItemText = (
+            <span>
+                {Utils.localizeMessage('forward_post_button.label', 'Forward')}
+                <Badge variant='success'>
+                    {Utils.localizeMessage('badge.label.new', 'NEW')}
```

Comment:
Is there any logic for when this should disappear? I'd assume this would go away at some point.

Also, this is a critique of the design, but I would've expected us to use a pulsing green dot for this sort of thing instead of the text NEW
![Screen Shot 2022-07-14 at 10 46 11 AM](https://user-images.githubusercontent.com/3277310/179010402-e191bddb-e327-4df2-813e-9bbb4f15f9a2.png)

Reply:
this was build as designed... I think the pulsating dot is more for places where there is not much space to actually show a badge.

Right now there is no logic to remove that. I suppose we will just remove the badge in a future dot release.

thoughts @esethna ?

Reply:
Usually that pulsating dot gets removed once you click on the feature for the first time

Reply:
yeah, I removed the comment because I thought those were your intentions, so I wasn't really adding anything.